definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -0,0 +1,34 @@
+# This list of inputs will replace any occurrence of `{Energy Storage System Type}`
+# in a variable-definition yaml file
+
+# This list is intended for use as input energy storage system to the power sector
+
+- Electricity Storage Type:
+
+  - Energy Storage System:
+      description: Mainly to refer to the batteries
+  - Energy Storage System|Lithium-Ion:
+      description: Lithium-Ion batteries
+  - Energy Storage System|Redox Flow:
+      description: Redox Flow batteries
+  - Energy Storage System|Compressed Air:
+      description: Compressed Air batteries
+
+  - Hydro:
+      description: hydropower
+  - Hydro|Run of River:
```

Comment:
Run of river is not a storage unit , it has no storage capacity so it should stay in the other tag